src/compressor/CMakeLists.txt

```diff
@@ -5,6 +5,13 @@ if (HAVE_QATZIP)
   list(APPEND compressor_srcs QatAccel.cc)
 endif()
 add_library(compressor_objs OBJECT ${compressor_srcs})
```

Comment:
@tchaikov the problem maybe found: the object library `compressor_objs` missed its dependencies. Here, we use `target_link_libraries()` to add the dependencies. And the line `include_directories("${QatDrv_INCLUDE_DIRS}") ` has been deleted.